packages/bsky/src/views/index.ts

```diff
@@ -63,8 +63,21 @@ export class Views {
   // Actor
   // ------------
 
+  actorIsNoHosted(did: string, state: HydrationState): boolean {
+    return (
+      this.actorIsDeactivated(did, state) || this.actorIsTakendown(did, state)
+    )
+  }
```

Comment:
I saw it defined elsewhere to take into account suspensions.  Do we need to do that here?  From `packages/bsky/src/hydration/actor.ts`:
```ts
      const isNoHosted =
        actor.takenDown ||
        ['takendown', 'suspended', 'deactivated'].includes(
          actor.upstreamStatus ?? '',
        )
```

Reply:
Ohp nevermind, I see it!